Components/GridPostsTeaser/_style.scss

```diff
@@ -0,0 +1,148 @@
+[is='flynt-grid-posts-teaser'] {
+  padding: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-tablet) {
+    padding: $component-spacing-tablet 0;
+  }
+
+  @media (min-width: $breakpoint-desktop) {
+    padding: $component-spacing-desktop 0;
+  }
+
+  .preContent {
+    @include reset-content-margins;
+
+    margin-bottom: 4.375rem;
```

Comment:
Looking at other components, the typical margin-bottom is 1.5rem or 50px. In GridPostsLatest we're using then 3rem, which is also weird (think that was me). We should unify this in general. So what can we agree on. 50px?

Reply:
The design uses mostly `70px` for this kind of spacing. If we are to follow the designer's recommendation I would then suggest using either the exact value, which is `4.375rem (70px)`.

The real question is: is there a particular reason to drop the original `70px` and go for `50px`?

![70px](https://user-images.githubusercontent.com/49150551/61519228-a7515c00-aa0b-11e9-8fa7-4821d88bf88d.png)